crowd_anki/export/note_sorter.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+
+from ..config.config_settings import ConfigSettings
+
+DEBUG = False
+
+@dataclass
+class NoteSorter():
+    sort_method: list
+    is_reversed: bool
+    skip_sorting: bool
+    sort_key_tuple: tuple
+
+    @classmethod
+    def from_config(cls, config: ConfigSettings):
+        sorting_definitions = {
+            ConfigSettings.DeckExportSortMethods.NO_SORTING: None,
+            ConfigSettings.DeckExportSortMethods.GUID: lambda i: i.anki_object.guid,
+            ConfigSettings.DeckExportSortMethods.FLAG: lambda i: i.anki_object.flags,
+            ConfigSettings.DeckExportSortMethods.TAG: lambda i: i.anki_object.tags,
+            ConfigSettings.DeckExportSortMethods.NOTEMODEL: lambda i: i.anki_object._model["name"],
+            ConfigSettings.DeckExportSortMethods.NOTEMODELID: lambda i: i.anki_object._model["crowdanki_uuid"],
+            ConfigSettings.DeckExportSortMethods.FIELD1: lambda i: i.anki_object.fields[0],
+            ConfigSettings.DeckExportSortMethods.FIELD2: lambda i: i.anki_object.fields[1]
+        }
+        
+        cls.sort_method = [ConfigSettings.DeckExportSortMethods._value2member_map_[method] for method in config.export_deck_sort_methods]
```

Comment:
I think the `config.export_deck_sort_methods` should return enum values straight away. Also the conversion to enum from value can be done by doing `DeckExportSortMethods(value)` afaik.   
Also I think it may make sense to make name and value the same for `DeckExportSortMethods`

Reply:
or you can make them case-insensetive and do `.upper` on all the strings